### PR TITLE
feat(orginization): add direct expiry date update api

### PR DESCRIPTION
**Pull request by Qi-Cui**
### Description

Fixes # (issue)

### Checklist

- [ ] 填写 PR 描述及相关 issue (write PR description and related issue)
- [ ] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [ ] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

File: src/bk-user/bkuser/apis/web/organization/views/users.py [hunk 1]
```diff
@@ -532,6 +533,44 @@ def delete(self, request, *args, **kwargs):
...
+        data = slz.validated_data
+
+        tenant_user.account_expired_at = data["account_expired_at"]
+        tenant_user.updater = request.user.username
+
+        # 根据租户用户当前状态判断，如果是过期状态则转为正常，如果是正常或停用则保持不变
+        if tenant_user.status == TenantUserStatus.EXPIRED:
+            tenant_user.status = TenantUserStatus.ENABLED
+
+        tenant_user.save(update_fields=["account_expired_at", "updater", "status", "updated_at"])
```
Comment (nannan00) [inline src/bk-user/bkuser/apis/web/organization/views/users.py:569]: Nit: 一般 关键更新字段放前面，对于 updater、updated_at 一般在最后

Comment (Qi-Cui) [reply src/bk-user/bkuser/apis/web/organization/views/users.py]: done

File: src/bk-user/bkuser/apis/web/organization/serializers/users.py [hunk 2]
```diff
@@ -311,7 +311,8 @@ def validate_account_expired_at(self, expired_at: datetime.datetime) -> datetime
         # Note: drf serializers.DateTimeField 会在 USE_TZ=True 时，
         #  将时间字符串 "2024-01-01 00:00:00" 直接附加当前用户的时区（不会进行时区转换），
         #  所以对于永久时间，只需要忽略时区或直接替换为 UTC 时区与常量（UTC时区）对比即可
-        if expired_at.replace(tzinfo=datetime.timezone.utc) == PERMANENT_TIME:
+        #  永久时间应为账号过期时间的最大值，所以对于大于永久的时间，则直接替换为永久时间
+        if expired_at.replace(tzinfo=datetime.timezone.utc) >= PERMANENT_TIME:
```
[the remark this answers is not in the document]
Comment (Qi-Cui) [reply src/bk-user/bkuser/apis/web/organization/serializers/users.py]: done

File: src/bk-user/bkuser/apis/web/organization/serializers/users.py [hunk 3]
```diff
@@ -328,6 +329,24 @@ def validate_account_expired_at(self, expired_at: datetime.datetime) -> datetime
         return expired_at
 
 
+class TenantUserExpiryDateUpdateInputSLZ(serializers.Serializer):
```
[the remark this answers is not in the document]
Comment (Qi-Cui) [reply src/bk-user/bkuser/apis/web/organization/serializers/users.py]: done

**Review by nannan00 — APPROVED** [review]

**Pull request merged**